resources/i18n/ca_ES/cura.po

```diff
@@ -0,0 +1,7245 @@
+
+#: /Users/j.delarago/development/cura_installation/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDeviceManager.py:481
+msgctxt "@title:window"
+msgid "Remove printers?"
+msgstr "Voleu eliminar les impressores?"
+
+#: /Users/j.delarago/development/cura_installation/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDeviceManager.py:484
+#, python-brace-format
+msgctxt "@label"
+msgid ""
+"You are about to remove {0} printer from Cura. This action cannot be undone.\n"
+"Are you sure you want to continue?"
+msgid_plural ""
+"You are about to remove {0} printers from Cura. This action cannot be undone.\n"
+"Are you sure you want to continue?"
+msgstr[0] ""
+"Esteu a punt d'eliminar {0} impressora del Cura Aquesta acció no es pot desfer.\n"
+"Esteu segur que voleu continuar?"
+msgstr[1] ""
+"Esteu a punt d'eliminar {0} impressores del Cura Aquesta acció no es pot desfer.\n"
+"Esteu segur que voleu continuar?"
+
+#: /Users/j.delarago/development/cura_installation/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDeviceManager.py:489
+msgctxt "@label"
+msgid ""
+"You are about to remove all printers from Cura. This action cannot be undone.\n"
+"Are you sure you want to continue?"
+msgstr ""
+"Esteu a punt d'eliminar totes les impressores del Cura. Aquesta acció no es pot desfer.\n"
+"Esteu segur que voleu continuar?"
+
+#: /Users/j.delarago/development/cura_installation/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDevice.py:154
+msgctxt "@action:button"
+msgid "Print via cloud"
+msgstr "Imprimeix a través del núvol"
+
+#: /Users/j.delarago/development/cura_installation/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDevice.py:155
+msgctxt "@properties:tooltip"
+msgid "Print via cloud"
+msgstr "Imprimeix a través del núvol"
+
+#: /Users/j.delarago/development/cura_installation/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDevice.py:156
+msgctxt "@info:status"
+msgid "Connected via cloud"
+msgstr "Connectat a través del núvol"
+
+#: /Users/j.delarago/development/cura_installation/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDevice.py:261
+msgctxt "@action:button"
+msgid "Monitor print"
+msgstr "Fes seguiment de la impressió"
+
+#: /Users/j.delarago/development/cura_installation/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDevice.py:263
+msgctxt "@action:tooltip"
+msgid "Track the print in Ultimaker Digital Factory"
+msgstr "Feu seguiment de la impressió a Ultimaker Digital Factory"
+
+#: /Users/j.delarago/development/cura_installation/Cura/plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDevice.py:279
+#, python-brace-format
+msgctxt "@error:send"
+msgid "Unknown error code when uploading print job: {0}"
+msgstr ""
+"S'ha produït un error de codi desconegut en carregar la tasca d'impressió: "
+"{0}"
+
+#: /Users/j.delarago/development/cura_installation/Cura/plugins/3MFWriter/__init__.py:28
+msgctxt "@item:inlistbox"
+msgid "3MF file"
+msgstr "fitxer 3MF"
```

Comment:
You couldn't have known this from the original text, but to be consistent it'd be good to use capitals here.
```suggestion
msgstr "Fitxer 3MF"
```